licences-specs/src/test/resources/GebConfig.groovy

```diff
@@ -26,8 +26,8 @@ driver = {
   // System.setProperty('webdriver.chrome.driver', '/Applications/chromedriver')
   // System.setProperty('webdriver.chrome.driver', '/usr/local/bin/chromedriver')
   ChromeOptions options = new ChromeOptions()
-  options.addArguments('headless')
-//  options.addArguments('chrome')
+ options.addArguments('headless')
+  // options.addArguments('chrome')
```

Comment:
guess this file should just be reverted?

Reply:
done